nxdrive/utils.py

```diff
@@ -1286,3 +1288,40 @@ def compute_digest(path: Path, digest_func: str, callback: Callable = None) -> s
         return UNACCESSIBLE_HASH
 
     return str(h.hexdigest())
+
+
+def save_config(conf_name: str, section: str, config_dump: Dict[str, Any]) -> Path:
```

Comment:
Remove the `conf_name` argument. The name is not meant to be customizable at all.